src/components/BoardOptionsMenu/BoardOptionsMenu.tsx

```diff
@@ -114,10 +116,12 @@ const BoardOptionsMenu = () => {
           dispatch(updateTitle(result.board.title));
           dispatch(updateAllLists(result.lists));
           dispatch(updateAllCards(result.cards));
-          // TODO add import for tags
+          dispatch(updateAllTags(result.tags));
+          // TODO add import for attachments
```

Comment:
Import and export of attachments is a bit unknown to me so I've created #174 to add it